API.md

```diff
@@ -45,9 +45,9 @@ The following errors include additional response properties:
 - `errno: 103`
 - `errno: 104` name
 - `errno: 105`
-- `errno: 106` address, bouncedAt, bouce
-- `errno: 107` address, bouncedAt, bouce
-- `errno: 108` address, bouncedAt, bouce
+- `errno: 106` address, bouncedAt, bounce
```

Comment:
It seems unfortunately that the "Complaint Error" still has a "bouncedAt" field, but perhaps we can't help that for API-compatibility reasons.